sp/src/materialsystem/stdshaders/shatteredglass.cpp

```diff
@@ -6,6 +6,7 @@
 // $NoKeywords: $
 //=============================================================================//
 
+#if 0
```

Comment:
Doesn't this disable the entire `SDK_ShatteredGlass` shader?

Reply:
Whoops, how did that get here? That's an error on my part, I'll fix it up :)